operator/github/v0/README.mdx

```diff
@@ -0,0 +1,266 @@
+| Token | `token` | string | Fill in your GitHub access token for advanced usages. For more information about how to create tokens, please refer to the https://github.com/settings/tokens. |
+
+
+
+
+## Supported Tasks
+
+### Get All Pull Requests
+
+Get the list of all pull requests in a repository
+
+
+| Input | ID | Type | Description |
+| :--- | :--- | :--- | :--- |
+| Task ID (required) | `task` | string | `TASK_GET_ALL_PULL_REQUESTS` |
+| Owner (required) | `owner` | string | owner of the repository |
+| Repository (required) | `repository` | string | repository name |
+| State (required) | `state` | string | State of the PRs, including open, closed, all. Default: open |
+| Sort (required) | `sort` | string | Sort the PRs by created, updated, popularity, or long-running. Default: created |
+| Direction (required) | `direction` | string | Direction of the sort, including asc or desc. Default: asc |
+
+
+
+| Output | ID | Type | Description |
+| :--- | :--- | :--- | :--- |
+| Pull Requests (optional) | `pull_requests` | array[object] | An array of PRs |
+
+
+
+
+
+
+### Get Pull Request
+
+Get a pull request from a repository, given the PR number. Default is the latest PR
+
+
+| Input | ID | Type | Description |
+| :--- | :--- | :--- | :--- |
+| Task ID (required) | `task` | string | `TASK_GET_PULL_REQUEST` |
+| Owner (required) | `owner` | string | owner of the repository |
+| Repository (required) | `repository` | string | repository name |
+| PR Number | `pr_number` | integer | Number of the PR. `0` for the latest PR |
+
+
+
+| Output | ID | Type | Description |
+| :--- | :--- | :--- | :--- |
+| Pull Request | `pull-request` | object | A pull request in GitHub |
+
+
+
+
+
+
+### Get Commit
+
+Get a commit from a repository, given the commit SHA
+
+
+| Input | ID | Type | Description |
+| :--- | :--- | :--- | :--- |
+| Task ID (required) | `task` | string | `TASK_GET_COMMIT` |
+| Owner (required) | `owner` | string | owner of the repository |
+| Repository (required) | `repository` | string | repository name |
+| Commit SHA (required) | `sha` | string | sha of the commit |
+
+
+
+| Output | ID | Type | Description |
+| :--- | :--- | :--- | :--- |
+| Commit | `commit` | object | A commit in GitHub |
+
+
+
+
+
+
+### Get Review Comments
+
+Get the review comments in a pull request
+
+
+| Input | ID | Type | Description |
+| :--- | :--- | :--- | :--- |
+| Task ID (required) | `task` | string | `TASK_GET_REVIEW_COMMENTS` |
+| Owner (required) | `owner` | string | owner of the repository |
+| Repository (required) | `repository` | string | repository name |
+| PR Number | `pr_number` | integer | Number of the PR. Default is the latest PR |
+| Sort | `sort` | string | Sort the comments by created, updated. Default: created |
+| Direction | `direction` | string | Direction of the sort, including asc or desc. Default: asc |
+| Since | `since` | string | Only comments updated at or after this time are returned. Default: 2021-01-01T00:00:00Z |
```

Comment:
```suggestion
| Sort | `sort` | string | Sort the comments by created, updated. Default is created |
| Direction | `direction` | string | Direction of the sort, including asc or desc. Default is asc |
| Since | `since` | string | Only comments updated at or after this time are returned. Default is 2021-01-01T00:00:00Z |
```